eng/pipelines/templates/jobs/smoke-tests.yml

```diff
@@ -0,0 +1,135 @@
+parameters:
+  - name: Daily
+    default: true
+  - name: Artifacts
+    type: object
+    default: []
+
+jobs:
+  - ${{ if eq(parameters.Daily, false) }}:
```

Comment:
Given the type is unconstrained and the default is `true`, this might be better as: `${{ if ne(parameters.Daily, true) }}`.

Reply:
I changed the type to bool, as I think the false equality check reads a little better.